docs/posts/ibis-duckdb-geospatial/index.qmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Ibis Geospatial: Introducing support for DuckDB spatial extension"
+author: Naty Clementi
+date: 2023-12-07
+categories:
+    - blog
+    - duckdb
+    - geospatial
+---
+
+Ibis now has support for some DuckDB geospatial functions (check the list [here](https://gist.github.com/ncclementi/fbc5564af709e2d7f8882821e3a8649f)).
+
+This blogpost showcases some examples of usage of the geospatial API for the DuckDB backend. The material is inspired in the ["DuckDB: Spatial Relationships"](https://geog-414.gishub.org/book/duckdb/07_spatial_relationships.html) lesson from [Dr. Qiusheng Wu](https://geog-414.gishub.org/book/preface/instructor.html)'s course "Spatial Data Management" from the Department of Geography & Sustainability at the University of Tennessee, Knoxville.
```

Comment:
Can you wrap all the long lines?

While it's easyish to review on GitHub, when someone else opens these in an editor they'll have to have soft wrap set.

While that might be fine, it's a bit more friendly to hard wrap.

Rendering will not change (only GitHub-flavored Markdown rendering is sensitive to wrapping).